http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/MonitorAtBoot.scala

```diff
@@ -144,11 +144,13 @@ class MonitorAtBoot[F[_]: Timer](publisherQueue: fs2.concurrent.Queue[F, LeoPubs
               )
           }
           diskIdOpt = app.appResources.disk.flatMap(d => if (d.status == DiskStatus.Creating) Some(d.id) else None)
+          diskNameOpt = app.appResources.disk.flatMap(d => if (d.status == DiskStatus.Creating) Some(d.name) else None)
           msg = CreateAppMessage(
             cluster.googleProject,
             action,
             app.id,
             app.appName,
+            diskNameOpt,
```

Comment:
why is disk `DiskName` needed to be passed in the message? I think `diskId` is already available from the message

Reply:
I'm not sure i understand how to get `diskName` from `diskId` - wouldn't that require querying the DB?

Reply:
yea..I think in subscriber, we're probably already querying for the disk anyway

Reply:
alright, looking into it!

Reply:
i think i've made the right change! please let me know what you think!